openid-sse-framework-1_0.txt

```diff
@@ -1504,21 +1617,182 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 26]
 
 
 
-Tulshibagwale, et al.   Expires December 10, 2021              [Page 27]
+
+
+
+
+Tulshibagwale, et al.        Standards Track                   [Page 29]
 
                           openid-sse-framework                 June 2021
 
 
-   +------+------------------------------------------------------------+
-   | Code | Description                                                |
-   +------+------------------------------------------------------------+
-   | 401  | if authorization failed or it is missing                   |
-   |      |                                                            |
-   | 403  | if the Event Receiver is not allowed to update the stream  |
-   |      | configuration                                              |
-   +------+------------------------------------------------------------+
+     +======+========================================================+
+     | Code | Description                                            |
+     +======+========================================================+
+     | 401  | if authorization failed or it is missing               |
+     +------+--------------------------------------------------------+
+     | 403  | if the Event Receiver is not allowed to read the       |
+     |      | stream status                                          |
+     +------+--------------------------------------------------------+
+     | 404  | if there is no Event Stream with the given "stream_id" |
+     |      | for this Event Receiver, or if the Subject specified   |
+     |      | is invalid or if the Receiver is not authorized to get |
+     |      | status for the specified Subject.                      |
+     +------+--------------------------------------------------------+
+
+                     Table 6: Read Stream Status Errors
+
+   Examples:
```

Comment:
These examples apply to all API calls. Should we move them to a part of the API description that applies to all calls?

Reply:
Yes. I think the examples sections all need a bit of rework. Some endpoints don't have examples, others (like this one) have examples that are too general. Since this was already in the spec, I'd like to leave it for a different pull request that is focused strictly on examples, if that's alright.

Reply:
See #37